pkg/storaged/fsys-panel.jsx

```diff
@@ -182,11 +168,12 @@ export class FilesystemsPanel extends React.Component {
                     className={mounts.length ? 'table-hover' : ''}
                     onRowClick={onRowClick}
                     columns={[
-                        { title: _("Name"), transforms: [cellWidth(30)], sortable: true },
-                        { title: _("Mount point"), transforms: [cellWidth(30)], sortable: true },
-                        { title:  _("Size"), transforms: [cellWidth(40)] }
+                        { title: _("Source"), transforms: [cellWidth(25)], sortable: true },
```

Comment:
I think these transforms don't work anymore with the new implementation. I forgot to port them to the 'new' way when I ported pkg/lib/cockpit-components-table.

Reply:
Right, they don't seem to make a difference.  Removed.